blog/2024-06-03-ratify-1.2.0.mdx

```diff
@@ -0,0 +1,68 @@
+
+## A brand new Cosign experience! 
+
+Starting in v1.2.0, Ratify now has more comprehensive Cosign verification capabilities. 
+
+Verification of Cosign signatures is a critical validation. As such, the Cosign verifier is now a built-in verifier along with the Notation verifier. Beyond first class support, the Cosign verifier introduces a `Trust Policy` which binds a particular set of verification settings to a certain registry scope. Now, it's possible to specify multiple trust policies based on your registry, namespace, and/or image. 
+
+Cosign verification also includes a robust key experience that integrates directly with KMP resources. Expanded capabilities include: multiple key support from KMP or filesystem and RSA + elliptical keys across multiple sizes.
+
+<!-- TODO: update links -->
+
+Watch this short demo to see the Cosign Verifier in action. Or check out the [documentation](https://ratify.dev/docs/reference/verifiers/cosign) for more information.
+
+[![asciicast](https://asciinema.org/a/658139.svg)](https://asciinema.org/a/658139)
+
+## Support for Multi-Tenancy in Kubernetes
+
+Ratify now includes multi-tenancy support. This new feature is designed to improve resource efficiency within Kubernetes clusters by enabling both multi-team and multi-customer architectures. 
+
+Multi-tenancy primarily focuses on a multi-team model, allowing organizations to effectively manage namespaces and allocate resources. Key user roles, such as cluster admins, team admins, and team developers, now have defined access and management capabilities.
+
+The addition of multi-tenancy support aims to facilitate seamless deployment, robust security, and efficient resource utilization, ensuring a smoother and more scalable experience for all.
+<!-- TODO: update links -->
+Watch this short demo to see multi-tenancy in action. Or check out the [documentation](https://ratify.dev/docs/reference/multi-tenancy) for more information.
+
+[![asciicast](https://asciinema.org/a/PO8RPcuZ2qyShif80c55WQipj.svg)](https://asciinema.org/a/PO8RPcuZ2qyShif80c55WQipj)
+
+## Introduction of the Key Management Provider
+
+In this release, we are introducing the KeyManagementProvider (KMP), a new resource designed to replace the existing CertificateStore. 
+
+The KMP resource is a comprehensive solution for managing and retrieving keys and certificates. It includes an inline provider, which allows for the specification of a single key or certificate. Additionally, it features an Azure Key Vault provider, which enables the fetching of any number of keys or certificates. 
+
+This new feature aims to streamline key and certificate management, enhancing the efficiency and security of your operations..
+
+## An invitation to contribute!
+
+We are thrilled to share that Ratify is currently under review to be adopted by the Cloud Native Computing Foundation (CNCF). 
```

Comment:
Do you want to add a link to the sandbox application link? https://github.com/cncf/sandbox/issues/96

Reply:
Excellent idea, I'll get that added. Thank you. :)